src/System.Console/tests/ConsoleEncoding.cs

```diff
@@ -11,6 +11,8 @@
 
 public partial class ConsoleEncoding : RemoteExecutorTestBase
 {
+    public static bool ConsoleEncodingAvailable => !PlatformDetection.IsUap || PlatformDetection.IsWindows10RS3OrGreater;
```

Comment:
@ViktorHofer someone has to suffer to find all the bugs before customers 😉 (I'll hunt for them on the VM though)